clang/test/Index/Core/scan-deps-cas.m

```diff
@@ -2,19 +2,19 @@
 
 // RUN: c-index-test core --scan-deps %S -output-dir=%t -cas-path %t/cas \
 // RUN:  -- %clang -c -I %S/Inputs/module \
-// RUN:     -fmodules -fmodules-cache-path=%t.mcp \
+// RUN:     -fmodules -fmodules-cache-path=%t.mcpit \
```

Comment:
`rm -rf %t.mcpit` at the beginning? Or just move both under `%t/mcp` and `%t/mcpit` subdirectories.

Reply:
Good catch! Moved to subdirectory.